### PR TITLE
Add configuration to build releases

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
- Modify Makefile (rename build by build-deps) to avoid failures during  promu crossbuild tarballs
- Modify promu config because of failure for 386 architecture
- Modify travis config to build and deploy releases on Github
- Renaming  .build by .build-deps because promu crossbuild tarballs will try to tar some files here and then fail at the end.

### CODE REVIEW

File: .promu.yml [hunk 1]
```diff
@@ -16,26 +16,27 @@ tarball:
...
         - docs/example.yaml
         - LICENSE
+
 crossbuild:
     platforms:
+# The build is failing for 386 architecture
+# It's commented until we fix it
         - linux/amd64
-        - linux/386
+#        - linux/386
```
Comment (iksaif) [inline .promu.yml]: just remove them

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 NAME    := unsee
 VERSION := $(shell git describe --tags --always --dirty='-dev')
-GO      := GO15VENDOREXPERIMENT=1 go
-PROMU   := $(GOPATH)/bin/promu
-pkgs     = $(shell $(GO) list ./... | grep -v -E '/vendor/')
+GO    := GO19VENDOREXPERIMENT=1 go
```
Comment (iksaif) [inline Makefile]: Are you sure this variable exists ?

File: Makefile [hunk 3]
```diff
@@ -29,44 +29,56 @@ endif
 
 .DEFAULT_GOAL := $(NAME)
 
-.build/deps-build-go.ok:
-	@mkdir -p .build
-	$(GO) get -u github.com/golang/dep/cmd/dep
-	$(GO) get -u github.com/jteeuwen/go-bindata/...
-	$(GO) get -u github.com/elazarl/go-bindata-assetfs/...
+.build-deps/deps-build-go.ok:
```
Comment (iksaif) [inline Makefile:32]: why .build-deps ?

Comment (CharlesJUDITH) [reply Makefile:32]: because if we keep .build, promu crossbuild tarballs will try to tar them and then fail at the end.

File: Makefile (hunk 2 above)
Comment (CharlesJUDITH) [reply Makefile]: not sure.

Comment (iksaif) [reply Makefile]: then why do you add it ?

File: Makefile (hunk 3 above)
Comment (iksaif) [reply Makefile:32]: Can you copy the promu in the changelog to make that more obvious ?

Comment (CharlesJUDITH) [reply Makefile:32]: I updated the PR. let me know if by changelog you meant commit message. Thx

Comment (iksaif) [reply Makefile:32]: Yep, commit message :)

**Pull request merged**